activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   PostgreSQL & MySQL: Use bigserial as primary key type for new tables
```

Comment:
We should say big integer since bigserial isn't a thing in MySQL and the serial aspect in postgres just means auto incrementing.